### PR TITLE
Feature/3 view achievements

### LINKED ISSUE #3: view-achievements
Making a way to view achievements.

### LINKED ISSUE #4: Unit Test Environment
Set up the unit testing environment within Django.

### PR DESCRIPTION
**Pull request by HansNorg**
Closes #3

### CODE REVIEW

File: squire/urls.py [hunk 1]
```diff
@@ -1,22 +1,11 @@
-"""base URL Configuration
+"""base URL Configuration"""
```
Comment (helaan) [inline squire/urls.py]: Why remove this comment?

File: achievements/templates/achievements/view-category.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{% extends 'achievements/base.html' %}
+
+{% block content %}
+  <h3>Name:</h3>
+  <p>{{name}}</p>
+
+  <h3>Description:</h3>
+  <p>{{description}}</p>
+{% endblock %}
```
Comment (helaan) [inline achievements/templates/achievements/view-category.html]: (Future) improvement: list achievements in this category

**Comment by helaan** [comment]
Base seems sensible, code not tested

File: squire/urls.py (hunk 1 above)
Comment (HansNorg) [reply squire/urls.py]: I don't know, probably should indeed not be removed.

File: achievements/templates/achievements/view-category.html (hunk 2 above)
Comment (HansNorg) [reply achievements/templates/achievements/view-category.html]: That is indeed something that I want to add in the next issue, where members will also be able to be linked to achievements and view their achievements

**Pull request merged**